README-Gitpod.md

```diff
@@ -0,0 +1,39 @@
+Token Contract in AssemblyScript - Gitpod version
+=================================================
+
+This is a stripped-down example where we'll demonstrate creating a token. We'll visit a page, sign in and use your browser's console to run a commands to initialize, send, and get the balance of a custom token.
+
+This README is specific to Gitpod and this example. For local development, please see [README.md](README.md).
+
+## Description
+
+This project contains implementation of token contract similar to [ERC20](https://theethereum.wiki/w/index.php/ERC20_Token_Standard).
```

Comment:
I think we could get rid of the "Description" heading and merge this statement with the main description under the first header

Reply:
I wonder if it makes sense for this to be the opening line of the description, rather than line 3. Also, I think it should probably say "contains _an_ implementation of _a_ token contract..."